content/news/playtest-20150531.md

```diff
@@ -0,0 +1,24 @@
+We have received a small number of issues after last week's playtest release. These issues have been addressed and so we are releasing a new playtest for you to check out!
+
+The list of fixes is agreeably short, which is to be expected after this equally short development cycle. The fixed issues include:
+
+* A number of issues pertaining to single-player missions in both Tiberian Dawn and Red Alert.
+* A crash in Dune 2000 that happened when a Deviator hit a harvester being picked up by a Carryall.
+* Rendering glitches which resulted in highly saturated range circles and blocky nuke explosion effects.
+
+For the full changelog, please refer to http://changelog.openra.net.
+
+The installers for a number of supported operating systems can be found on [the download page](/download/) as usual.
+
+If you encounter any problems or bugs, don't hesitate to report them [on our bug tracker](http//bugs.openra.net).
```

Comment:
> If you encounter any problems...

I am not sure if we should advertise the bug tracker as a support ticket system for general questions even though our forums suck a lot.